pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,10 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+func init() {
+	metrics.Register()
```

Comment:
This is an improvement, but metrics itself should protect against double-registration IMO.